valhalla/odin/directionsbuilder.h

```diff
@@ -18,42 +16,40 @@ namespace odin {
  */
 class DirectionsBuilder {
 public:
-  DirectionsBuilder();
```

Comment:
i have no idea why this object was being instantiated. there was actually no state at all in any of the class so i changed it to be all static so that, from the outside it looks like what it is, a collection of static functions.